tests/Functional/Action/Lti/UpdateLtiOutcomeActionTest.php

```diff
@@ -72,30 +77,28 @@ public function testItReturns401IfWrongAuthentication(): void
 
         $this->kernelBrowser->request(
             'POST',
-            '/api/v1/lti/outcome? '. $queryParameters,
+            '/api/v1/lti/outcome? ' . $queryParameters,
```

Comment:
There's a space between the question mark and querystring parameter. is this correct?

Reply:
No it's not, nice catch :)